testing/simulated/transformers.go

```diff
@@ -96,7 +96,7 @@ func transformExecutableDataToExecutionPayload(
 	data *gethprimitives.ExecutableData,
 ) (*ctypes.ExecutionPayload, error) {
 	// Check that the fork version is supported (pre-Deneb1).
-	if !version.IsBefore(forkVersion, version.Deneb1()) {
+	if version.EqualsOrIsAfter(forkVersion, version.Deneb1()) {
```

Comment:
nit: comment is off. IIUC here we are changing the versions range we're testing (from pre Deneb1 to post or equal postDeneb1)

Reply:
I missed the negation. All good

Reply:
nit but this would read easier if said
```suggestion
	if version.IsAfter(forkVersion, version.Deneb()) {
```